src/battle/item/dizzy_dial/dizzy_dial.c

```diff
@@ -0,0 +1,51 @@
+#include "dizzy_dial.h"
+
+Script N(main) = SCRIPT({
+    SI_VAR(10) = c 0x9A;
```

Comment:
This should be `= c ItemID_DizzyDial` I'm pretty sure.
Same for other items - obviously different item IDs, but they should exist in `enums.h`.